src/style.css

```diff
@@ -2,20 +2,18 @@ body {
   margin: 0;
 }
 
-.red {
-  background-color: red;
+div {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice